### DISCUSSION TITLE
Volta Font Editing - Eliminated MS4

### DISCUSSION THREAD

**Discussion by EricFulHat** [musescore/MuseScore#15694] [General]
n Musescore 3, the inspector was able to edit the style of volta texts - italics, font size, etc...

This seems to have been eliminated in Musescore 4 - when clicking on the Volta and going to properties - text, it no longer has these options.

In the attached files, you will see that my formatting for the volta from MS3 is retained. However, proceeding forward with MS4, I am unable to edit this to achieve italics and smaller font size. This is also generally true for Staff Text options with the line and a few other features.

[6th INCOMPLETE - Table.pdf](https://github.com/musescore/MuseScore/files/10335076/6th.INCOMPLETE.-.Table.pdf)

**Comment by ghost** [comment]
This is a known issue, for which a design and task has been created: https://github.com/musescore/MuseScore/issues/14238
